From: Kill Your TV <killyourtv@i2pmail.org>
Date: Wed, 4 Feb 2015 15:38:20 +0000
Subject: another FTBFS fix

Since I2PControl is not used yet, I'll comment out the offending
section.
---
 I2PControl.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/I2PControl.cpp b/I2PControl.cpp
index e90e08e..936db86 100644
--- a/I2PControl.cpp
+++ b/I2PControl.cpp
@@ -111,9 +111,11 @@ namespace client
 	void I2PControlService::ReadRequest (std::shared_ptr<boost::asio::ip::tcp::socket> socket)
 	{
 		auto request = std::make_shared<I2PControlBuffer>();
+		/*
 		socket->async_read_some (boost::asio::buffer (*request),                
 			std::bind(&I2PControlService::HandleRequestReceived, this, 
 			std::placeholders::_1, std::placeholders::_2, socket, request));
+		*/
 	}
 
 	void I2PControlService::HandleRequestReceived (const boost::system::error_code& ecode,
